examples/base_cc_gwlb_asg/variables.tf

```diff
@@ -161,7 +161,7 @@ variable "health_check_interval" {
 variable "healthy_threshold" {
   type        = number
   description = "The number of successful health checks required before an unhealthy target becomes healthy. Minimum 2 and maximum 10"
-  default     = 3
+  default     = 2
```

Comment:
@vkrishnamurthy-zscaler  this isn't intended for asg deployments as they would be destroyed when unhealthy. This is not the case for non-asg gwlb deployments (current production though).

Case would be smedge upgrade with 30 second intervals now where http probe may very well go unhealthy for a short time. This will bring it back up faster as it would require one less interval check of successful 200 OKs

Reply:
Fine.  My misunderstanding or imagination. I thought maybe autoscale(gwlb to be specific )sends a heartbeat to unhealthy instances but kind of rollbacks the health state if the health check starts passing again.